lib/fog/compute/google/models/disk.rb

```diff
@@ -74,11 +74,15 @@ def attached_disk_obj(opts = {})
         end
 
         def get_as_boot_disk(writable = true, auto_delete = false)
+          get_disk(writable, true, auto_delete)
+        end
+
+        def get_disk(writable = true, boot = false, auto_delete = false)
           {
-            :auto_delete => auto_delete,
-            :boot => true,
+            :autoDelete => auto_delete,
```

Comment:
`auto_delete` and `autoDelete` should be interchangeable in `google-api-client`.

Or does `auto_delete` not function correctly anymore?

Reply:
Good catch! Changed to `auto_delete `.